src/effects/Wahwah.cpp

```diff
@@ -60,24 +51,144 @@ const ComponentInterfaceSymbol EffectWahwah::Symbol
 
 namespace{ BuiltinEffectsModule::Registration< EffectWahwah > reg; }
 
-BEGIN_EVENT_TABLE(EffectWahwah, wxEvtHandler)
```

Comment:
The elimination of this table, and the use of BindTo calls instead, could be done at step 2 of the transformation.